### DISCUSSION TITLE
Page action button icon center

### DISCUSSION THREAD

**Discussion by jwhoami** [filamentphp/filament#3621] [Help] [answered]
Hi,

Instead of add text to the button, I prefer to just have the icon and add a tooltip for further help.
I can remove the icon using label(false) on the button itself, however, visually, the icon is not centered perfectly.

Right now the button looks like this (icon pulls to the left)

![i2](https://user-images.githubusercontent.com/74318608/186028000-ec1e31e8-9a81-4ae7-b564-7a0dd6640dad.png)

I would like it to look like this (icon centered)

![i1](https://user-images.githubusercontent.com/74318608/186028060-c3784550-5ee2-4915-8c57-a37e169387b1.png)

Thanks

**Comment by mikrosmile** [comment, marked as answer]
```
->iconButton()
```

**Reply by jwhoami** [reply]
Thanks @mikrosmile. I think its not yet documented.

However, just to make the answer a bit more concrete for someone in my situation, I got it working like so...

      Actions\CreateAction::make()
        ->iconButton()
        ->icon('heroicon-o-plus')
        ->label(false)
        ->toolTip(__('Create'))
        ->extraAttributes(['class' => 'rounded-lg border text-white bg-primary-600 hover:bg-primary-500 focus:bg-primary-700 focus:ring-offset-primary-700']),
`
I looked up the classes on a regular page action.

**Reply by mikrosmile** [reply]
In case of using iconButton you can remove label(false) at all

**Reply by danharrin** [reply]
We have `disableLabel()` if you want to remove the label spacing without `iconButton()`, keeping the background color, while preserving accessibility for screenreaders
